### PR TITLE
chore: add volunteer Aryan-Deshpande

### PR DESCRIPTION
**Pull request by Aryan-Deshpande**

### CODE REVIEW

File: data/users/Aryan-Deshpande.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Aryan Deshpande",
+    "bio": "Backend Engineer",
+    "image": "https://discord.com/channels/835750351621718027/835750351621718030/1047868354569973791",
```
Comment (codewithvoid) [inline data/users/Aryan-Deshpande.json:4]: This is not loading. Can you use GitHub or Twitter profile pic url

File: data/users/Aryan-Deshpande.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["NodeJS", "Flask", "Javascript","Docker", "Kubernetes", "Azure", "AWS S3"]
```
Comment (codewithvoid) [inline data/users/Aryan-Deshpande.json:8]: Keep each entry max 2-3

Comment (codewithvoid) [inline data/users/Aryan-Deshpande.json:8]:
```json
"Docker", "Kubernetes", "Azure"
```
Should come in devops to be added as part of #89

Comment (Aryan-Deshpande) [reply data/users/Aryan-Deshpande.json:8]: made the required changes, should I include the slug "devops" in my file changes ( json file ) ?

Comment (codewithvoid) [reply data/users/Aryan-Deshpande.json:8]: sure, i have added it already in #92

**Comment by codewithvoid** [comment]
Thanks for making the changes :)

**Pull request merged**